sklearn/ensemble/iforest.py

```diff
@@ -217,7 +216,7 @@ def fit(self, X, y=None, sample_weight=None):
             self._contamination = self.contamination
 
         if self.behaviour == 'old':
-            warnings.warn('behaviour="old" is deprecated and will be removed '
+            warn('behaviour="old" is deprecated and will be removed '
```

Comment:
actually I think you need to fix the indentation here. make sure you run flake8 on this file